swift_browser_ui_frontend/src/common/lang.js

```diff
@@ -77,8 +77,7 @@ let default_translations = {
         sharing_to_one_project: "This folder is shared to one project.",
         sharing_to_many_projects: "This folder is shared to multiple projects.",
         shared_with_read:
-          "This folder is shared with you. You can view files only by" +
-          " using SD Desktop and download files in this folder (Read access)",
+          "This folder is shared with you.",
```

Comment:
Does this text also related to SD-Connect branding thing so that's why it was removed? 
I found this text from [Figma](https://www.figma.com/proto/N0Y3N0nfyMS2wG3gjwZ201/SD-Connect-Prototype?node-id=2094%3A6510&scaling=min-zoom&page-id=517%3A2532&starting-point-node-id=517%3A2533&show-proto-sidebar=1), but I'm not sure if it's the latest version.

Reply:
Actually the term "SD Desktop" is also removed from Figma. I agree that we can recheck later if some texts should be put in here :)